inst/extdata/meta/colour_scales.csv

```diff
@@ -0,0 +1,8 @@
+country,indicator,colour,min,max,invert_scale
```

Comment:
where are these scales (interpolateViridis, interpolateMagma, etc) actually living? What is the full set of possibilities?

Reply:
Scales are from https://github.com/d3/d3-scale-chromatic#api-reference